doc/userguide/rules/multi-buffer-matching.rst

```diff
@@ -90,3 +90,4 @@ following keywords:
 * ``quic.cyu.string``
 * ``tls.certs``
 * ``tls.cert_subject``
+* ``tls.subjectaltname``
```

Comment:
This is a nit but I find this string difficult to read. We use `_` in other sticky buffer names, perhaps `tls.subject_altname` (much easier to read, imo).